app/src/main/java/knu/dong/teamproject/adapter/ChatbotListAdapter.kt

```diff
@@ -0,0 +1,32 @@
+package knu.dong.teamproject.adapter
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.BaseAdapter
+import knu.dong.teamproject.databinding.ListviewItemChatBinding
+import knu.dong.teamproject.databinding.ListviewItemRoleBinding
+import knu.dong.teamproject.dto.Chatbot
+
+class ChatbotListAdapter(val context: Context, val chatList: MutableList<Chatbot>) : BaseAdapter() {
+    override fun getCount(): Int {
+        return chatList.size
+    }
+
+    override fun getItem(position: Int): Any {
+        return chatList[position]
+    }
+
+    override fun getItemId(position: Int): Long {
+        return 0
+    }
```

Comment:
```suggestion
    override fun getItemId(position: Int): Long {
        return position.toLong()
    }
```

position에 해당되는 아이템의 id를 추출하는 함수라 이런식으로 구현해주어야 함.